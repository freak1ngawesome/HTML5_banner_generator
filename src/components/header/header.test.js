import React from 'react';
import Header from './header';
import renderer from 'react-test-renderer';

describe('Testing <Header/>', () => {
  it('Component has rendered correctly',() => {
    const header = renderer.create(<Header/>).toJSON();
    expect(header).toMatchSnapshot();
  });
});
