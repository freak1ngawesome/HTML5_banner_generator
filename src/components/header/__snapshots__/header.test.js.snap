// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Header/> Component has rendered correctly 1`] = `
<header
  className="header"
>
  <h1
    className="header__heading"
  >
    Create your awesome banner
  </h1>
</header>
`;
